config/config.go

```diff
@@ -825,8 +825,8 @@ func NewConfig() (c *Config) {
 					},
 				},
 				I18n: I18nUIDefaults{
-					Default:   "en",
-					Languages: []string{"en"},
+					Language:     "en",
+					ShowSelector: true,
```

Comment:
I think the default should be `ShowSelector: false`, right?   We don't want to show the selector when most users will just want English.  Only Chinese users will maybe want to see the selector.